src/main/java/org/opensearch/sdk/ExtensionSettings.java

```diff
@@ -25,30 +25,25 @@ public class ExtensionSettings {
      */
     public static final String EXTENSION_DESCRIPTOR = "src/test/resources/extension.yml";
```

Comment:
Just noticed that this path is in `src/test/resources` so this constant really shouldn't be under `src/main/java`.  Not sure we need to move it now as this is placeholder code, but where will we eventually be getting this?   

May want to make it package private at least.

Reply:
This is just a placeholder file. Once we integrate SDK with any extension, this will point to the extenions.yml of the extensions

Reply:
Lets open an issue to track this problem. 
When we ship the SDK customers cannot use this, it should be a file path which is passed to `ExtensionsRunner`

Reply:
https://github.com/opensearch-project/opensearch-sdk/issues/52